migration/migration.go

```diff
@@ -743,14 +736,12 @@ func createOrUpdatePlannerItemExtension(ctx context.Context, typeID uuid.UUID, n
 func createOrUpdateType(ctx context.Context, typeID uuid.UUID, spaceID uuid.UUID, name string, description string, extendedTypeID *uuid.UUID, fields map[string]workitem.FieldDefinition, icon string, witr *workitem.GormWorkItemTypeRepository, db *gorm.DB) error {
 	log.Info(ctx, nil, "Creating or updating planner item types...")
 	wit, err := witr.LoadTypeFromDB(ctx, typeID)
-	cause := errs.Cause(err)
-	switch cause.(type) {
-	case errors.NotFoundError:
+	if ok, _ := errors.IsNotFoundError(err); ok {
 		_, err := witr.Create(ctx, spaceID, &typeID, extendedTypeID, name, &description, icon, fields)
 		if err != nil {
 			return errs.WithStack(err)
 		}
-	case nil:
```

Comment:
in this case, we just do it when error is nil which is good. However we also do nothing if we get an internal server error from the Load operation